lib/datadog/appsec/processor.rb

```diff
@@ -94,6 +80,30 @@ def finalize
 
       private
 
+      def update_rules_data_with_static_configured_values
```

Comment:
Thinking we might want that to be more general, something like:

```
merge_to_rule_data('blocked_ips' => Datadog::AppSec.settings.ip_denylist, 'blocked_users' => Datadog::AppSec.settings.user_id_denylist)
```

And a companion:

```
def apply_denylist(settings)
  update_rule_data(merge_to_rule_data('blocked_ips' => settings.ip_denylist, 'blocked_users' => settings.user_id_denylist))
end

# usage: apply_denylist(Datadog::AppSec.settings)
```

WDYT?

Reply:
I like the suggestion. 

One thing that I do not think is necessary right now is the `merge_to_rule_data` method. If we see that the mreing logic gets complicated or that we use it in other places of the codebase we could consider creating the method and exposing it as part of the Processor API. 

But for now I think this is readable enough:

```ruby
def apply_denylist_data(settings)
  ruledata_setting = []
  ruledata_setting << denylist_data('blocked_ips', settings.ip_denylist)
  ruledata_setting << denylist_data('blocked_users', settings.user_id_denylist)

  update_rule_data(ruledata_setting)
end
```

Reply:
Fair enough